lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
+        scope = scope || 'global';
+        return new Promise(function(resolve, reject) {
```

Comment:
You can just do:

```
return waterline.templates.destroy(...);
```

Rather than wrapping it in a Promise constructor, since all the waterline model methods are already promisified.